### DISCUSSION TITLE
Detect Transcription Factors

### DISCUSSION THREAD

**Discussion by VerisimilitudeX** [VerisimilitudeX/DNAnalyzer#225] [Ideas]
https://en.wikipedia.org/wiki/Transcription_factor

Please post below if anyone has ideas about how we could go about detecting transcription factors in DNA. We would greatly appreciate your help!

**Comment by Speedro** [comment]
Hi @Verisimilitude11 I would like to have a look at this one, if that's possible. 🚀

**Comment by VerisimilitudeX** [comment]
Sure @Speedro. I actually need some time to do some research about transcription factors in order to update the description. If you want to get started earlier, feel free to research this using the Wikipedia article above.

**Comment by Speedro** [comment]
Fine, thanks, I will go through the wiki article, but would be great if there is some further description. I also wanted to ask how does it look with the unit tests. I believe that would be really beneficial to introduce them and as the gradle is already adopted, we could include the junit pretty easily.

**Comment by VerisimilitudeX** [comment]
I think that for unit tests, we can just use junit to verify PRs for now. I'll add more to the description soon (I need to expand by genetics knowledge as well). Feel free to implement any additional ideas that you may have for testing.
